.gitignore

```diff
@@ -66,3 +66,4 @@ captures/
 !/.idea/encodings.xml
 !/.idea/copyright/
 !/.idea/compiler.xml
+app/libs
```

Comment:
? what do you have in app/libs? You should probably delete this

Reply:
@macgills We've the latest version(that contains kiwix-serve) of kiwix-lib locally. 
We have to do this until kiwix-lib's new version is published on **Bintray**.

Reply:
the gitignore changes don't need to be pushed though, just have them local

Reply:
Fixed. fc4ba04544a69ac85388a2c70642ea51f6b13d14